generator/templates/rust/rust.jinja2

```diff
@@ -0,0 +1,29 @@
+{% import "rust/constant.jinja2" as constant %}
+{# {% import "cpp/enumeration.jinja2" as enumeration %} #}
+{% import "rust/structure.jinja2" as structure %}
+{% import "rust/archive.jinja2" as archive %}
+// Do not edit: This code was generated by flatdata's generator.
+
+use flatdata::Struct;
+
+{% macro render_namespaces(root) %}
+{% for module in root.children if module | is_namespace %}
```

Comment:
What if we do not have any namespace?

Reply:
Not having a root namespace is an invalid schema. I checked.